.travis.yml

```diff
@@ -6,15 +6,22 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
-# env:
-#   - GROONGA_MASTER=yes
+env:
+  matrix:
+    - GROONGA_MASTER=no GROONGA_LATEST=yes
+    - GROONGA_MASTER=yes GROONGA_LATEST=no
+    - GROONGA_MASTER=no GROONGA_LATEST=no
```

Comment:
Should we run these patterns for all Rubies? The auto built Groonga isn't depended on Ruby version.

```diff
diff --git a/.travis.yml b/.travis.yml
index 4b19b2ff..84cc25f0 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -2,19 +2,25 @@ sudo: required
 notifications:
   webhooks:
     - https://webhook.commit-email.info/
-rvm:
-  - 2.4
-  - 2.5
-  - 2.6
 # env:
 #   - GROONGA_MASTER=yes
-# matrix:
-#   allow_failures:
-#     - rvm: ruby-head
+matrix:
+  - name: "Ruby 2.4"
+    rvm: 2.4
+  - name: "Ruby 2.5"
+    rvm: 2.5
+  - name: "Ruby 2.6"
+    rvm: 2.6
+  - name: "Auto Groonga build"
+    rvm: 2.6
+    env:
+      - INSTALL_GROONGA=no
 before_install:
-  - curl --silent --location https://github.com/groonga/groonga/raw/master/data/travis/setup.sh | sh
   - |
-    if [ "${GROONGA_MASTER}" != "yes" ]; then
-      sudo apt-get -qq -y install groonga-tokenizer-mecab
+    if [ "${INSTALL_GROONGA}" != "no" ]; then
+      curl --silent --location https://github.com/groonga/groonga/raw/master/data/travis/setup.sh | sh
+      if [ "${GROONGA_MASTER}" != "yes" ]; then
+        sudo apt-get -qq -y install groonga-tokenizer-mecab
+      fi
     fi
   - gem update bundler
```